starter-files/05-login.html

```diff
@@ -6,7 +6,7 @@ <h1 class="flex justify-center flex-row w-full leading-loose text-3xl">
     <div class="w-full max-w-xs">
       <form class="bg-blue-darker shadow-md rounded px-8 pt-6 pb-8 mb-4">
         <div class="inline-block relative w-64 mt-2">
-          <label class="sr-only" for="select-user">Select a user</label>
+          <label class="sr-only">Select a user</label>
```

Comment:
I'm purposefully removing the `for` attribute so that it gets flagged as an a11y violation. It's also made consistent with the examples in the notes.